### ISSUE TITLE
questions about the visualization 

### ISSUE THREAD

**Issue by ghost** [xwjabc/hed#7]
Thank you for this elegent project !  I am trying to train a HED model using my own dataset. All the hyper-parameters are followed by this project and the results is also reasonable. However, the fifth side outputs of all the test image (shown below) are almost gray, which confuses me greatly. Could you have some ideas about this scenario?

![batch-499-1st-image](https://user-images.githubusercontent.com/24312469/51987652-f3502900-24dd-11e9-9e9e-0b85d5918ca4.png)
 
![batch-499-1st-image](https://user-images.githubusercontent.com/24312469/51987609-dca9d200-24dd-11e9-9e87-5e9ae009f3dd.png)
![batch-499-1st-image](https://user-images.githubusercontent.com/24312469/51987623-e5020d00-24dd-11e9-9e79-8ee6a6476e84.png)

**Comment by xwjabc** [comment]
Interesting. I did not see this situation before. Perhaps you can do the following experiments:
1) Train the original HED in your environment and see if the 5th side is gray.
2) Use pre-trained HED on your dataset and see if the 5th side is gray.
I wonder if the scenario is due to the training process or the environment setting.

**Comment by xiaopengguo1406** [comment]
Thank you for these valuable suggestions. I have conducted corresponding experiments to observe the situation. Both of the  5th side are not gray. 
Also, I changed the learning rate of conv5.weight and conv5.bias from args.lr*100, args.lr*200 to args.lr*10, args.lr*20, respectively, and all the other parameters remain unchanged. Surprisedly, the 5th sides are not gray and the fused results become reasonable. Hence, I guess the original learning rate is relatively large for my dataset, resulting in a gray results of 5th side.

Finally, thank you again for this elegent project, which is very easy to run and has a clear instruction.

**Comment by CoachingJane** [comment]
hello! when you train your own dataset did you use pretrain model? if you have used, which pretrained model didi you use :5stage-vgg.caffemodel? or epoch-19-checkpoint.pt? or hed_pretrained_bsds.caffemodel? thank you!
